include/aos/common/config.hpp

```diff
@@ -127,6 +127,146 @@
 #define AOS_CONFIG_TYPES_FILE_CHUNK_SIZE 1024
 #endif
 
+/**
+ * File system mount destination len.
+ */
+#ifndef AOS_CONFIG_TYPES_FS_MOUNT_DESTINATION_LEN
+#define AOS_CONFIG_TYPES_FS_MOUNT_DESTINATION_LEN 48
```

Comment:
Probably we should be more general here, please provide(if none) smth like cFSPathLen & utilize it for 
AOS_CONFIG_TYPES_FS_MOUNT_SOURCE_LEN / DESTINATION